docs/integrations/sources/firebolt.md

```diff
@@ -58,4 +58,5 @@ You can now use the Airbyte Firebolt source.
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
-| 0.1.0 | 2022-04-28 | TBD | Create Firebolt source |
\ No newline at end of file
+| 0.2.0 | 2022-09-09 | TBD | Reading from views |
```

Comment:
Update this too.

Reply:
yep, updated the doc.  I am going  to run test again. Can you help remove the docker image before I run the publish?